### PR TITLE
feat: add Scroll Back to Top button for better navigation on Roadmap …

### PR DESCRIPTION
**Pull request by devXpraAddy**
https://github.com/user-attachments/assets/39d1d3de-c5a0-407d-ac73-6cbdc419fbdb

### CODE REVIEW

File: package-lock.json
Comment (offensive-vk) [inline package-lock.json:1]: Remove the `package-lock.json` changes.

**Review by offensive-vk — COMMENTED** [review]
cc @arikchakma .

File: package-lock.json
Comment (devXpraAddy) [reply package-lock.json:1]: done

**Comment by kamranahmedse** [comment]
Please open an issue with feature request and get an approval before you make such changes. 

https://github.com/kamranahmedse/developer-roadmap/pull/7413/files

**Pull request closed without merging**